### PR TITLE
WIP: Organization for the next meeting on 2018/09/13

### PR DESCRIPTION
**Pull request by amilan**
Initial commit with the agenda for the meeting to be held on 2018/09/13
at 14.00

### CODE REVIEW

File: 20180913/AGENDA.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Sardana Follow-up Meeting - 2018/09/13
+
+To be held on 2018/09/13 at 14:00
+
+# Agenda
+
+1. Presentation
+    * Any volunteer?
```
Comment (reszelaz) [inline 20180913/AGENDA.md]: Maybe, together with @daneos - what do you think?, we could make a short presentation about the release process steps. When I say presentation, just go through the [release process steps](https://github.com/sardana-org/sardana/blob/develop/doc/how_to_release.md). present our impressions about the last release, etc.

IMHO the step from AOB deserves to be moved to the 1st point:)

File: 20180913/AGENDA.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+3. Collaborative Developments - Review
+
+4. New Developments, Enhancements or Bug Fixes
+
+5. Pending PR distribution
+
+6. Any Other Business
+    * Sardana Documentation Camp summary.
+    * Brainstorming on how to improve and organize work.
```
Comment (reszelaz) [inline 20180913/AGENDA.md:20]: It would be nice if, before the meeting, all of us could think of few ideas on how to improve and organize our work.

**Review by reszelaz — COMMENTED** [review]
Thanks @amilan for proposing the meeting. And sorry about not comenting before. Just two minor comments from my side. Thanks again and see you tomorrow!

File: 20180913/AGENDA.md (hunk 1 above)
Comment (daneos) [reply 20180913/AGENDA.md]: Sure, we can move this to be the first point.

**Comment by amilan** [comment]
Hi,
I have uploaded the my minutes from the meeting. 
If you have time, give them a look and let me know if there is anything missing or that needs to be changed.

Cheers,
Antonio.

**Comment by reszelaz** [comment]
Minues are ok! I also uploaded the slides. Thanks!

**Comment by reszelaz** [comment]
What about merging this one? Anyone else to comment on minutes?

**Comment by daneos** [comment]
Minutes look good. I think it can be merged.

**Review by daneos — APPROVED** [review]

**Comment by teresanunez** [comment]
For me it can also be merged. Only one question, what means the 'eas' in:

"Next release done by Teresa or Antonio and share eas."

A typo?

**Comment by reszelaz** [comment]
> what means the 'eas' in:

I don't remember. If @amilan neither remembers I propose to say something like:

> Rotative process. Next release done by Teresa or Antonio with help/guidance of Grzegorz or Zibi.

**Comment by amilan** [comment]
Hi all, 

I'm back from holidays (today is the first day).
I think the 'eas' is just a typo and should be: ideas. I think I remember that we said that we will make it rotative and we will share ideas about possible improvements.
In any case, I'm going to change that with @reszelaz suggestion. 
After that, I think we could merge it.

**Comment by amilan** [comment]
There we go. 
There were a couple of extra typos ... also fixed now.

Cheers.

**Comment by reszelaz** [comment]
Great! Thansk @amilan and welcome back:)

**Pull request merged**